omise/payment_methods/true_money.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . 'omise/payment_methods/_offsite.php';
+}
+
+class OmisePaymentMethod_TrueMoney extends OmiseOffsitePaymentMethod
+{
+
+    const
+        NAME = 'TrueMoney',
+        PAYMENT_OPTION_NAME = 'omise-truemoney-payment',
+        DEFAULT_TITLE = 'True Money',
+        TEMPLATE = 'truemoney_payment',
+        SWITCH_DESCRIPTION = 'Enables payments by True Money (currently only available in Thailand).'
+    ;
+
```

Comment:
there is one extra breakline here

Reply:
Intentional

Reply:
Ah no - it was off here. It is already addressed in a newer PR